man/fmidi-play.1

```diff
@@ -0,0 +1,34 @@
+.TH FMIDI\-PLAY "1" "August 2021" "" "User Commands"
+.SH NAME
+fmidi\-play \(en MIDI player with a terminal interface
+.SH SYNOPSIS
+.B fmidi\-play
+.RI [ OPTION \ ...]
+.I INPUT
+.RI [ INPUT \ ...]
+.P
+Plays the specified MIDI file or files with an interactive terminal interface.
+.SH OPTIONS
+.TP
+.B \-r
+Play
+.I INPUT
+files in random order.
+.TP
+.B \-n\fR\ \fINAME
+Specify MIDI client name (default
+.RB \(lq fmidi \(rq).
+.TP
+.B \-M\ \fR[\fBalsa\fR|\fBjack\fR]
+Choose a particular MIDI API for playback; default is unspecified, allowing the
+RtMidi library to choose.
```

Comment:
I prefer RtMidi not mentioned explicitly, this is implementation detail and might change.